src/main/java/games/cantstop/CantStopGameState.java

```diff
@@ -97,7 +98,10 @@ protected CantStopGameState _copy(int playerId) {
 
     @Override
     protected double _getHeuristicScore(int playerId) {
-        return getGameScore(playerId);
+        if (!isNotTerminal()) {
```

Comment:
Shouldn't this be reversed? not+not = yes. so if (terminal) return game score, else return result (which should always be 0)? Maybe that's intended, just a double-check.